doc/howto/instances_configure.md

```diff
@@ -31,15 +32,25 @@ Others are updated only when the instance is restarted.
 See the "Live update" column in the {ref}`instance-options` table for information about which options are applied immediately while the instance is running.
 ```
 
+(instances-configure-devices)=
 ## Configure devices
 
 To add and configure an instance device for your instance, use the `lxc config device add` command.
-Specify the instance name, a device name, the device type and maybe device options (depending on the {ref}`device type <device-types>`):
+Devices can be added or removed while the instance is running.
```

Comment:
Sadly this isn't true for all device types when using VMs.

Reply:
Here's the run down:

The general rule is:

- Containers can hot plug.
- VMs cannot hot plug.

Exceptions:

- `disk` can hot plug for both containers and VMs.
- `gpu-mig` cannot hot plug.
- `nic-bridged` can hot plug for both containers and VMs.
- `nic-ipvlan` cannot hot plug.
- `nic-macvlan` can hot plug for both containers and VMs.
- `nic-p2p` can hot plug for both containers and VMs.
- `nic-phyical` can hot plug for both containers and VMs.
- `nic-routed` can hot plug for both containers and VMs.
- `nic-sriov` can hot plug for both containers and VMs.
- `proxy` can hot plug for both containers and VMs.
- `usb` can hot plug for both containers and VMs.

Reply:
OK, added the general information here and the more specific info to the devices pages (I'll clean that up in a follow-up PR).